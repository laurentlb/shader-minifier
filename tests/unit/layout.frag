#version 330

// #486
layout(       location   =  1  )     out vec4 f_color2;

// #17
layout(location=0) out vec4 f_color;
//subroutine float functionType(int i);
//layout(location = 0) subroutine uniform functionType tbl[2];
//layout(index = 1) subroutine (functionType) float func1(int i) { return i * 1.0/255.0; }
//layout(index = 2) subroutine (functionType) float func2(int i) { return 1.0 - i * 1.0/255.0; }
