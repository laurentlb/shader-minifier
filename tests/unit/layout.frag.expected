#version 330

layout(location=1)out vec4 f_color2;
layout(location=0)out vec4 f_color;
